packages/android_intent_plus/example/android/app/src/main/java/io/flutter/plugins/androidintentexample/MainActivity.java

```diff
@@ -1,17 +1,22 @@
 package io.flutter.plugins.androidintentexample;
 
+import android.annotation.SuppressLint;
 import android.content.IntentFilter;
 import android.os.Bundle;
+
+import androidx.core.content.ContextCompat;
+
 import io.flutter.embedding.android.FlutterActivity;
 
 public class MainActivity extends FlutterActivity {
 
-  @Override
-  protected void onCreate(Bundle savedInstanceState) {
-    super.onCreate(savedInstanceState);
+    @SuppressLint("WrongConstant")
```

Comment:
Here is the same thing: https://github.com/fluttercommunity/plus_plugins/pull/1811/files#r1193861724

Looks like Google folks don't really care about fixing this bug.